crt_portal/cts_forms/templates/forms/report_data.html

```diff
@@ -0,0 +1,163 @@
+<div class="crt-portal-card">
+    <div class="crt-portal-card__content margin-bottom-3">
+        {% include "forms/snippets/preview_button.html" with step_name="contact" step_value=0 %}
```

Comment:
Can we use `step_name=ordered_step_names.0` here like we do in the other cards below?